web/templates/pages/list_access_key.html

```diff
@@ -0,0 +1,110 @@
+<!-- Begin toolbar -->
+<div class="l-center edit">
+    <div class="l-sort clearfix">
+        <div class="l-unit-toolbar__buttonstrip">
+            <a class="ui-button cancel" dir="ltr" id="btn-back" href="/list/access-key/"><i
+                        class="fas fa-arrow-left status-icon blue"></i><?= _('Back'); ?></a>
+        </div>
+    </div>
+</div>
+<!-- End toolbar -->
+
+<div class="l-separator"></div>
+
+<div class="l-center animated fadeIn">
+
+    <form id="vstobjects">
+        <table class="data mode-add">
+            <tr class="data-add">
+                <td class="data-dotted">
+                    <table class="data-col1">
+                        <tr>
+                            <td></td>
+                        </tr>
+                    </table>
+                </td>
+                <td class="data-dotted">
+                    <table class="data-col2">
+                        <tr>
+                            <td class="step-top">
+                                <span class="page-title"><?= _("Access Key") ?></span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <?php show_error_panel($_SESSION); ?>
+                            </td>
+                        </tr>
+
+                        <?php if (!empty($key_data['ACCESS_KEY_ID'])) { ?>
+                            <tr>
+                                <td class="vst-text step-top">
+                                    <?= _('Access Key Id'); ?>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>
+                                    <input type="text" size="20" class="vst-input" maxlength="255" disabled
+                                           value="<?= htmlentities(trim($key_data['ACCESS_KEY_ID'], "'")) ?>">
+                                </td>
+                            </tr>
+                        <?php } ?>
+
+                        <?php
+                        if (!empty($_SESSION['ok_msg'])) {
+                            ?>
+                            <?php if (!empty($key_data['ACCESS_KEY_ID']) && !empty($key_data['SECRET_ACCESS_KEY'])) { ?>
+                                <tr>
+                                    <td class="vst-text input-label">
+                                        <?= _('Secret Key'); ?><br>
+                                        <span style="font-size: 10pt; color:#ffd500;"><?= _('WARNING_TO_SAVE_ACCESS_KEY'); ?></span>
```

Comment:
<img width="482" alt="Screenshot 2022-04-18 at 19 10 08" src="https://user-images.githubusercontent.com/9754650/163845333-6fae246b-66d1-4108-abe9-91dc941fde35.png">

With the dark theme it is hard to read we should probally change the color slightly